org.zenframework.z8.server/src/main/java/org/zenframework/z8/server/base/table/system/Roles.java

```diff
@@ -79,6 +87,8 @@ public CLASS(IObject container) {
 			setJavaClass(Roles.class);
 			setName(Roles.TableName);
 			setDisplayName(displayNames.Title);
+			Optional.ofNullable(apiAttrs.Title)
+					.ifPresent(attrVal -> setAttribute("APIDescription", attrVal));

```

Comment:
Write without lambda (->), declare APIDescription constant in org.zenframework.z8.server.json.Json